main.tf

```diff
@@ -17,7 +17,7 @@ resource "aws_instance" "win_builder" {
   vpc_security_group_ids = [join("", aws_security_group.winrm_sg.*.id)]
 
   tags = {
-    Name = "${local.resource_name}-builder"
+    Name = "${local.resource_name}-builder-${local.ami_underlying[local.win_builder_ami_key]}"
```

Comment:
Why remove this?

Reply:
It's not removed, just updating the name tag with more info to make it easier to see what is building in the console.

Reply:
Oh, yes. Sorry confused the removing with the adding!